vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+- name: check for available disk space
+  become: yes
+  command: df -h /opt/
```

Comment:
here you want to use `{{ root_folder }}` vs `/opt`.  Sometimes `root_folder` is /opt, but more often it is `/opt/app` and on FedRAMP sites, `/opt/app` is its own file system.